common/validity/script.xml

```diff
@@ -35,12 +35,12 @@
 			Vaii Vith
 			Wara Wcho
 			Xpeo Xsux
-			Yezi Yiii Zanb
```

Comment:
what's going on with `Zanb`? @markusicu @macchiati

Reply:
We've been here before.. https://github.com/unicode-org/cldr/pull/3233#issuecomment-1701848571 

blocked by CLDR-17056  - I'll back out `Zanb` for now.